code/game/objects/structures/stool_bed_chair_nest/stools.dm

```diff
@@ -74,3 +73,17 @@
 		T.Weaken(5)
 		return
 	..()
+
+/obj/item/weapon/stool/dropped(mob/user) //Make sure we aren't leaving un-deconstructible stools.
+	..()
+	if(!user.in_throw_mode) //If this stool isn't getting thrown, turn it back into a structure.

```

Comment:
If they're dropping it while throw mode is set, not even throwing it, it'll remain an item.